docs/use_cases/recSys-basic.ipynb

```diff
@@ -0,0 +1,3606 @@
+      "\n",
+      "United Bans Many Popular Dog And Cat Breeds From Cargo Holds After Pet Deaths The airline unveiled new restrictions in its pet transportation policy.\n",
+      "\n",
+      "Why There Are Tiny Holes At The Bottom Of Windows On Planes If you’ve ever been on a plane, chances are, you’ve probably looked out your window, only to notice a tiny hole at the bottom\n",
+      "\n",
+      "These Are The Most Expensive Travel Days Of The Year We all love a good travel deal, so avoid these two days. There are two days of the year that are the absolute most expensive\n",
+      "\n",
+      "Roller Coaster Riders Suspended 100 Feet In The Air, Facing Down, After Malfunction An \"abnormality\" halted the coaster at the worst possible time.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print_articles_from_list(specific_articles)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 147,
+   "id": "e685700a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'TRAVEL'}"
+      ]
+     },
+     "execution_count": 147,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "set([news_articles.loc[id_]['category'] for id_ in specific_articles])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d87030b4",
+   "metadata": {},
+   "source": [
+    "It seems we have done it correctly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "id": "36a3d016",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#interactions = interactions.sample(n=600)\n",
+    "#interactions.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "id": "6dbe6c1f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#interactions.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c6b5986",
+   "metadata": {},
+   "source": [
+    "Let's create train and test data. The train data would be used for models training. \n",
+    "\n",
+    "We will provide two models. For the first, the train data would be \n",
+    "used to create a vector, for each user, where this vector would be populated by ratings given to news articles.\n",
+    "Once we have created the user vectors, we can retrieve the most similar users via, e.g., cosine similarity. \n",
+    "And once similar users are identified, we can easily collect items they have seen that the context users has not\n",
+    "yet seen. Let's call the first model \"Similar Vectors\".\n",
+    "\n",
+    "The second model is a Matrix Factorization model presented in [this paper](http://yifanhu.net/PUB/cf.pdf), with an efficient implementation in `implicit` package available [here](https://github.com/benfred/implicit)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "406ecb4e",
+   "metadata": {},
+   "source": [
+    "### Cold-start problem\n",
```

Comment:
I think it would be nice to point out that different recommendation policies can start to work at different levels of interaction numbers:
- 0 interactions -> most popular stuff, random stuff
- some interactions -> (personalized) content based
- more interactions -> collaborative filtering

Reply:
I added this in the "Cold-start" section, I think it makes sense to have it there